docs/resources/Channel.md

```diff
@@ -6,41 +6,41 @@ Represents a guild or DM channel within Discord.
+| flags?                              | integer                                                                     | [channel flags](#DOCS_RESOURCES_CHANNEL/channel-object-channel-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field)                                                       | GUILD_TEXT, GUILD_VOICE, GUILD_CATEGORY, GUILD_ANNOUNCEMENT, ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD, GUILD_STAGE_VOICE, GUILD_DIRECTORY, FORUM_CHANNEL |
+| total_message_sent?                 | integer                                                                     | number of messages ever sent in a thread, it's similar to `message_count` on message creation, but will not decrement the number when a message is deleted                                    | ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD                                                                                                                 |
+| available_tags?                     | array of [tag](#DOCS_RESOURCES_CHANNEL/forum-tag-object) objects            | the set of tags that can be used in a `GUILD_FORUM` channel                                                                                                                                   | GUILD_FORUM                                                                                                                                                        |
+| applied_tags?                       | array of snowflakes                                                         | the IDs of the set of tags that have been applied to a thread in a `GUILD_FORUM` channel                                                                                                      | PUBLIC_THREAD                                                                                                                                                      |
+| default_reaction_emoji?             | ?[default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object | the emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                                           | FORUM_CHANNEL                                                                                                                                                      |
+| default_thread_rate_limit_per_user? | integer                                                                     | the initial `rate_limit_per_user` to set on newly created threads in a channel. this field is copied to the thread at creation time and does not live update.                                 | GUILD_TEXT, GUILD_ANNOUNCEMENT, ANNOUNCEMENT_THREAD, PUBLIC_THREAD, PRIVATE_THREAD, FORUM_CHANNEL                                                                  |
```

Comment:
```suggestion
| default_thread_rate_limit_per_user? | integer                                                                     | the initial `rate_limit_per_user` to set on newly created threads in a channel. this field is copied to the thread at creation time and does not live update.                                 | GUILD_TEXT, GUILD_ANNOUNCEMENT, FORUM_CHANNEL                                                                  |
```

Reply:
The thread rate limit parameter is copied into thread objects.

Reply:
It's copied to `rate_limit_per_user`.